### PR TITLE
Support PostCSS 8

### PR DESCRIPTION
**Pull request by Semigradsky**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,121 +1,117 @@
...
+module.exports = () => ({
+  postcssPlugin: 'postcss-media-minmax',
+  AtRule(rule) {
+    if (rule.name !== "media" && rule.name !== "custom-media") {
+      return
+    }
 
-  }
+    transform(rule)
+  },
```
Comment (hudochenkov) [inline index.js:114]: [Migration guide](https://evilmartians.com/chronicles/postcss-8-plugin-migration#step-3-take-the-most-out-of-the-new-api) suggest subscribing to a specific names for better performance.

```js
  AtRule {
    media: (atRule) => {
      transform(atRule);
    },
    'custom-media': (atRule) => {
      transform(atRule);
    },
  },
```

Comment (Semigradsky) [reply index.js]: Updated

Comment (yisibl) [reply index.js]: @ai Can you help me see if there are other performance issues?

File: index.js [hunk 2]
```diff
@@ -1,121 +1,118 @@
...
-            var equals_for_max = $6;
-
-            if ($2 === '>') {
-              equals_for_min = $6;
-              equals_for_max = $3;
-            }
-
-            return create_query($4, '>', equals_for_min, min) + ' and ' + create_query($4, '<', equals_for_max, max);
-          }
+  rule.params = rule.params.replace(/\(\s*([a-z-]+?)\s*([<>])(=?)\s*((?:-?\d*\.?(?:\s*\/?\s*)?\d+[a-z]*)?)\s*\)/gi, function($0, $1, $2, $3, $4) {
```
Comment (ai) [inline index.js:62]:
```suggestion
  if (!atRule.params.includes('<') && !atRule.params.includes('>')) return

  rule.params = rule.params.replace(/\(\s*([a-z-]+?)\s*([<>])(=?)\s*((?:-?\d*\.?(?:\s*\/?\s*)?\d+[a-z]*)?)\s*\)/gi, function($0, $1, $2, $3, $4) {
```

A quick check will avoid calling the heavy parser for at-rules without any `<` and `>`.

Comment (yisibl) [reply index.js:62]: Thanks!

Comment (Semigradsky) [reply index.js:66]: Updated

File: index.js [hunk 3]
```diff
@@ -1,121 +1,122 @@
...
+    if (feature_name.indexOf($1) > -1) {
+      return create_query($1, $2, $3, $4);
+    }
+    //If it is not the specified attribute, don't replace
```
Comment (yisibl) [inline index.js:70]: Please add spaces for comments(`// If`).

**Review by yisibl — CHANGES_REQUESTED** [review]

File: index.js (hunk 3 above)
Comment (Semigradsky) [reply index.js]: Done

Comment (yisibl) [reply index.js]: Thanks!

**Comment by Semigradsky** [comment]
Release?

**Comment by Semigradsky** [comment]
@yisibl sorry, I published new version to npm, but I can't permissions to push changes. Commit - https://github.com/Semigradsky/postcss-media-minmax/commit/d0fe43c069e5ce484c1cc4d317a001c1cae136c6

**Comment by yisibl** [comment]
Thank you for your contribution. I updated the document. https://github.com/postcss/postcss-media-minmax/commit/97c9af06c6848b99604134c519806915e28ae650

**Pull request merged**